.github/workflows/build.yml

```
@@ -82,23 +81,28 @@ jobs:
         python3-nbsphinx
 
     - name: Install homebrew dependencies
-      if: matrix.os == 'macos-12'
+      if: ${{ contains(matrix.os, 'macos') }}
       run: |
-        brew install ccache gcc@12 llvm boost fftw hdf5 open-mpi openblas
+        brew update
+        brew install ccache gcc llvm boost fftw hdf5 open-mpi openblas
         mkdir $HOME/.venv
         python3 -m venv $HOME/.venv/my_python
         source $HOME/.venv/my_python/bin/activate
         pip install mako numpy scipy mpi4py
         pip install -r requirements.txt
         echo "VIRTUAL_ENV=$VIRTUAL_ENV" >> $GITHUB_ENV
-        echo "PATH=$PATH" >> $GITHUB_ENV
+        echo "PATH=$(brew --prefix llvm)/bin:$(brew --prefix gcc)/bin:$PATH" >> $GITHUB_ENV
 
-    - name: add clang cxxflags
+    - name: Add clang CXXFLAGS
       if: ${{ contains(matrix.cxx, 'clang') }}
       run: |
-        echo "PATH=/usr/local/opt/llvm/bin:$PATH" >> $GITHUB_ENV
         echo "CXXFLAGS=-stdlib=libc++" >> $GITHUB_ENV
 
+    - name: Add clang LDFLAGS for macos to link against brew's libc++
+      if: ${{ contains(matrix.os, 'macos') && contains(matrix.cxx, 'clang') }}
+      run: |
+        echo 'LDFLAGS="-L$(brew --prefix llvm)/lib/c++ -L$(brew --prefix llvm)/lib -lunwind"' >> $GITHUB_ENV
```

Single quotes will suppress expansion, this resulting in `LDFLAGS` being set literally to `-L$(brew --prefix llvm)/lib/c++ -L$(brew --prefix llvm)/lib -lunwind`, resulting in [errors](https://github.com/hmenke/w2dynamics_interface/actions/runs/13858838401/job/38781921730#step:10:201) like
```
ld: warning: search path '$(brew --prefix llvm)/lib/c++ -L$(brew --prefix llvm)/lib -lunwind' not found
```

```suggestion
        echo "LDFLAGS=\"-L$(brew --prefix llvm)/lib/c++ -L$(brew --prefix llvm)/lib -lunwind\"" >> $GITHUB_ENV
```

---

Turns out, linking to libunwind also [doesn't work](https://github.com/hmenke/w2dynamics_interface/actions/runs/13859514589/job/38784227486#step:9:50).
```
    ld: library 'unwind' not found
```

```suggestion
        echo "LDFLAGS=-L$(brew --prefix llvm)/lib/c++ -L$(brew --prefix llvm)/lib" >> $GITHUB_ENV
```

---

Hi Henri,

Thanks, this is definitely not correct. I don't remember why I put the LDFLAGS there in the first place. It seems to be working just fine without it. Have you tried removing it altogether?